### DISCUSSION TITLE
Can I follow the same trace I started in the client application on the server?

### DISCUSSION THREAD

**Discussion by cnovelli** [uptrace/uptrace#109] [Q&A]
what I want to do is to put in the same trace the tracking I do in the client application with the one I do in the server, can I collect the id of the trace or something similar so that it does not create a new trace with a different id? 

![Captura de pantalla de 2022-10-03 14-37-16](https://user-images.githubusercontent.com/100791325/193578242-6131478a-5a86-479c-8fa6-acd96028bf01.png)

I want to add under biuwer_signin_success another trace that instead of coming from the client application comes from the server application.
My programming language is js

**Comment by vmihailenco** [comment]
This is called [context propagation](https://uptrace.dev/opentelemetry/distributed-tracing.html#context-propagation). The details depend on the library you are using, but you can try https://github.com/open-telemetry/opentelemetry-js/discussions/2209
